23/README.md

```diff
@@ -0,0 +1,283 @@
+    --peer http://10.240.0.4:9984\
+    --peer http://10.240.0.5:9984\
+    --peer http://10.240.0.6:9984\
+    --peer http://10.240.0.7:9984\
+    send -r1000000 --mode=sync
+```
+
+This command uses 32 workers to push 1,000,000 transaction of 765 bytes to the four nodes in the network. The generated `CSV` file with the results was then copied to another machine
+
+### Experiment 2: finalize 1,000,000 transactions (experimental parallelization)
+For each node in the network, we started BigchainDB with `bigchaindb start --experimental-parallel-validation`. After some seconds, we switched to the workload generating virtual machine, and ran:
+
+```
+bigchaindb-benchmark\
+    --csv results/exp-32proc-1000000sync.csv\
+    --processes=32\
+    --peer http://10.240.0.4:9984\
+    --peer http://10.240.0.5:9984\
+    --peer http://10.240.0.6:9984\
+    --peer http://10.240.0.7:9984\
+    send -r1000000 --mode=sync
+```
+
+This command uses 32 workers to push 1,000,000 transaction of 765 bytes to the four nodes in the network. The generated `CSV` file with the results was then copied to another machine
+
+### Experiment 3: finalize 16,000 transactions (experimental parallelization)
+For each node in the network, we started BigchainDB with `bigchaindb start --experimental-parallel-validation` and we changed the flag `skip_timeout_commit` to `true` in `${HOME}/.tendermint/config/config.toml` (we noticed a significant delay to create the last block of the test, and skipping the timeout commit seems to help in this regard). After some seconds, we switched to the workload generating virtual machine, and ran:
+
+```
+bigchaindb-benchmark\
+    --csv results/exp-32proc-1000000async.csv\
+    --processes=32\
+    --peer http://10.240.0.4:9984\
+    --peer http://10.240.0.5:9984\
+    --peer http://10.240.0.6:9984\
+    --peer http://10.240.0.7:9984\
+    send -r16000 --mode=sync
+```
+This command uses 32 workers to push 16,000 transaction of 765 bytes to the four nodes in the network. The generated `CSV` file with the results was then copied to another machine
+
+## Results
+Each experiment generated a CSV file containing the following columns:
+- `txid`: the ID of the transaction being sent to the server
+- `size`: the size in bytes of the transaction
+- `ts_send`: timestamp when the transaction was sent.
+- `ts_accept`: timestamp when the transaction was accepted.
+- `ts_commit`: timestamp when the transaction was committed and finalized.
+- `ts_error`: timestamp of an error if it occurred.
+
+We built a script called [`analysis.py`][bdb:analysis.py] to process those CSV files and output four different graphs per experiment:
+- Distribution of how much time it takes to **accept** a transaction.
+- Distribution of how much time it takes to **finalize** a transaction.
+- **Accepted** transactions per second.
+- **Finalized** transactions per second.
+
+
+### Experiment 1: finalize 1,000,000 transactions (master)
+![Experiment 1][self:plot-experiment-1]
```

Comment:
GitHub renders the SVG file quite small, and clicking on it doesn't help. There doesn't seem to be any easy way to zoom in, even though in theory it is a vector graphic and therefore infinitely zoom-in-able. Maybe we could just embed a big JPEG for GitHub readers and provide the SVG download link if people want that.

Reply:
I've improved the SVG files, take a look and tell me what you think.

Reply:
I checked and the SVG files _do_ look much better now. I can now read the labels. Issue resolved. Disaster averted.